.github/workflows/r_solaris.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     name: solaris-cran
     timeout-minutes: 120
     runs-on: ubuntu-latest
-    container: rocker/r-base
+    container: wch1/r-debug
```

Comment:
just noting this is the same image used in sanitizer jobs:

https://github.com/microsoft/LightGBM/blob/da98f24711a2faab17f94e5b2a636e6609c93fa6/.github/workflows/r_package.yml#L175-L179

And those jobs are successfully building the package with vignettes.

https://github.com/microsoft/LightGBM/runs/4086356142?check_suite_focus=true

more evidence that this image has pandoc:

```shell
docker run --entrypoint="" -it wch1/r-debug pandoc --version

# pandoc 2.5
# Compiled with pandoc-types 1.17.5.4, texmath 0.11.2.2, skylighting 0.7.7
# Default user data directory: /root/.pandoc
```